src/WatchDogs.Infrastructure.FakeSource/IWatcher.cs

```diff
@@ -1,5 +1,5 @@
 ﻿
-namespace WatchDogs.FakeSource;
+namespace WatchDogs.Infrastructure.FakeSource;
```

Comment:
Why is the interface in Fake?

Reply:
There are also Interfaces in Watchdogs.Infrastructures.DxTrade and it was approved. This fact tamper to simply put interfaces directly related to WatchDogs.Infrastructure.FakeSource there

Reply:
Okay now it is more clear.